README.md

```diff
@@ -56,7 +76,7 @@ Let's quickly review an example where we implement a credit card payment scenari
 > For more examples covering almost all use-cases, check out the [examples in the `tests/` folder](./tests)
 
 ```go
-client, _ := craftgate.New("<YOUR API KEY>", "<YOUR SECRET KEY>", "https://sandbox-api.craftgate.io");
+client, _ := craftgate.New("<YOUR API KEY>", "<YOUR SECRET KEY>", "https://sandbox-api.craftgate.io", make(map[string]string));
```

Comment:
overload edip, map yerine sadece  lang alsak olmaz mı. Diğer clientlardaki standart ile aynı olması için.

Reply:
go overload method desteklemiyor abi. map yerine lang alabiliriz ama

Reply:
client options bunun için değil mi abi, şu örnekteki gibi with ile yapılabilir gibi geldi
https://stackoverflow.com/questions/56178203/multiple-constructors-in-go-optional-parameters

Reply:
bunu denedik ama olmadı, tekrar bakalım

Reply:
ekledim 🙏